### ISSUE TITLE
Add website section for organization biographies

### ISSUE THREAD

**Issue by nickschurch** [UK-NABES/UK-NABES.github.io#5]
This is important for evidencing that a network exists and for people to be able to see which organizations are involved. Need to have this in advance of the next workshop meeting (2024-02-14) which will be walking people through using GitHub and contributing to this site.

**Comment by nickschurch** [comment]
I think they're after a page with 1-2 paragraph descriptions of each organization, probably with a logo, maybe a main contact email...

**Comment by nickschurch** [comment]
Yep. I think ideally we'd just set it up so they can write some markdown for their org and commit it (along with an icon) and the autobuild just adds it without us having to do anything (a bit like the blog posts). That way if/when more people join it's no sweat. Same for individuals biographies.

**Comment by dill** [comment]
Cool, so this and the individuals pages just need to be the scaffolding for them to add content, that's no problem.

________________________________________
